api/src/controllers/search.js

```diff
@@ -419,6 +419,68 @@ let searchCollection = async function(
     }
   });
 
+  // For read only users, we need to redact out the details
+  // of any individual where the birthdate is null or the individual
+  // is less then 19 years old. First step to do this is calculate their
+  // age
+  if (roles.includes('public')) {
```

Comment:
I think `.includes` needs tweaking.  Public users will only have the `public` role, so the existence of such doesn't necessarily mean they are a public user.  Roles could look like:

`read: ['public', 'sysadmin']`

I think we should do the opposite logic:  Redact unless they include one of `exports.ApplicationRoles` in [misc.js](https://github.com/bcgov/NRPTI/blob/34b82aba6bacea0e1110a216c75b9ab68228b3a5/api/src/utils/constants/misc.js)

Reply:
Yeah sounds reasonable. I'll make a change.

Reply:
Updated!